---
title: '&#39;Schließlich&#39; kann nicht angezeigt werden, außerhalb einer &#39;versuchen&#39; Anweisung'
ms.date: 07/20/2015
f1_keywords:
- vbc30382
- bc30382
helpviewer_keywords:
- BC30382
ms.assetid: 0314d8d2-18bc-4bbd-858c-0a18408b52fd
ms.openlocfilehash: 0e16604aed715884df7ca4aa52716804603b7533
ms.sourcegitcommit: 3d5d33f384eeba41b2dff79d096f47ccc8d8f03d
ms.translationtype: MT
ms.contentlocale: de-DE
ms.lasthandoff: 05/04/2018
ms.locfileid: "33613186"
---
# <a name="39finally39-cannot-appear-outside-a-39try39-statement"></a>&#39;Schließlich&#39; kann nicht angezeigt werden, außerhalb einer &#39;versuchen&#39; Anweisung
`Finally` wird zum Beenden eines `Try...Catch...Finally` -Blocks verwendet, kann daher nur einmal am Ende des Blocks angegeben werden. Entweder ist `Finally`unnötig, oder die `Finally` -Anweisung befindet sich außerhalb des entsprechenden `Try` -Blocks.  
  
 **Fehler-ID:** BC30382  
  
## <a name="to-correct-this-error"></a>So beheben Sie diesen Fehler  
  
1.  Suchen und entfernen Sie die unnötige `Finally` Anweisung.  
  
2.  Verschieben Sie die `Finally` -Anweisung an die gewünschte Position im Code.  
  
## <a name="see-also"></a>Siehe auch  
 [Try...Catch...Finally-Anweisung](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)  
 
